app/util/session_id.py

```diff
@@ -0,0 +1,16 @@
+import uuid
+
+
+class SessionId(object):
+    SESSION_HEADER_KEY = 'Session-Id'
```

Comment:
I saw that the convention for header keys was 'First-Letter-Uppercase-With-Dashes', and just went with that.